http/server.go

```diff
@@ -290,13 +305,26 @@ func (s *Server) getQuoteHandler(w http.ResponseWriter, r *http.Request) {
 	dec := json.NewDecoder(r.Body)
 	dec.DisallowUnknownFields()
 	err := dec.Decode(&qr)
+
 	if err != nil {
 		log.Error("error decoding request: ", err.Error())
 		http.Error(w, "bad request", http.StatusBadRequest)
 		return
 	}
 	log.Debug("received quote request: ", fmt.Sprintf("%+v", qr))
 
+	maxValueTotransfer := s.cfgData.MaxQuoteValue
+
+	if maxValueTotransfer <= 0 {
+		maxValueTotransfer = s.cfgData.RSK.MaxQuoteValue
+	}
+
+	if qr.ValueToTransfer.Uint64() > maxValueTotransfer {
+		log.Error("error on quote value, cannot be greater than: ", s.cfgData.MaxQuoteValue)
+		http.Error(w, "internal server error", http.StatusInternalServerError)
```

Comment:
maybe we should have a better msg here? I mean, I dont know if "internal server error" would be reasonable for when valueToTransfer is higher than the maximum allowed.

Maybe something like "server error: value to trransfer cant be higher than maximum allowed"?